doc/intro.rst

```diff
@@ -0,0 +1,119 @@
+<https://hyperspy.org/>`_, these and other domain specific features have been 
+split out into dedicated packages with the :external+hyperspy:ref:`HyperSpy 2.0 
+release <changes_2.0>`.
+
+Check out the :ref:`installation` section for further information, including
+how to start using this project.
+
+.. note::
+
+   This is an open-source community project: everyone is welcome to contribute.
+   Please read our (see :ref:`contributing_label`) guidelines and get started!
+
+
+Learning resources
+==================
+
+.. grid:: 2 3 3 3
+  :gutter: 2
+
+  .. grid-item-card::
+    :link: user_guide/installation
+    :link-type: doc
+
+    :octicon:`rocket;2em;sd-text-info` Getting Started
+    ^^^
+
+    New to holoSpy or the HyperSpy ecosystem? The getting started guide provides an
+    introduction on basic usage of holoSpy and how to install it.
+
+  .. grid-item-card::
+    :link: user_guide/index
+    :link-type: doc
+
+    :octicon:`book;2em;sd-text-info` User Guide
+    ^^^
+
+    The user guide provides in-depth information on key concepts of holoSpy
+    and how to use it along with background information and explanations.
+
+  .. grid-item-card::
+    :link: api/index
+    :link-type: doc
+
+    :octicon:`code-square;2em;sd-text-info` Reference
+    ^^^
+
+    Documentation of the metadata specification and of the Application Progamming Interface (API),
+    which describe how holoSpy functions work and which parameters can be used.
+
+  .. grid-item-card::
+    :link: https://github.com/hyperspy/holospy-demos
+
+    :octicon:`workflow;2em;sd-text-info` Tutorials
+    ^^^
+
+    Tutorials in form of Jupyter Notebooks to learn how to process
+    off-axis electron holography data using holoSpy.
+
+  .. grid-item-card::
+    :link: contributing
+    :link-type: doc
+
+    :octicon:`people;2em;sd-text-info` Contributing
+    ^^^
+
+    holoSpy and the whole HyperSpy ecosystem are a community project
+    maintained for and by its users. There are many ways you can help!
+
+  .. grid-item-card::
+    :link: changes
+    :link-type: doc
+
+    :octicon:`zap;2em;sd-text-info` Changes
+    ^^^
+
+    To learn about recent changes, check out the release notes.
+
+
+Citing HoloSpy
+==============
+
+HoloSpy is maintained by the `HyperSpy community
+<https://github.com/hyperspy/holospy/contributors>`_.
+
+If HoloSpy has been significant to a project that leads to an academic
+publication, please acknowledge that fact by citing it. The DOI in the
+badge below is the `Concept DOI <https://help.zenodo.org/faq/#versioning>`_ --
+it can be used to cite the project without referring to a specific
+version. If you are citing HoloSpy because you have used it to process data,
+please use the DOI of the specific version that you have employed. You can
+find it by clicking on the DOI badge below:
+
+.. image:: https://zenodo.org/badge/doi/10.5281/zenodo.10137993.svg
+    :class: sd-bg-transparent
```

Comment:
Same comments as in https://github.com/hyperspy/exspy/pull/36#discussion_r1545590838.

Reply:
Without this line, I see a thin gray line on the edge of the badge in dark mode. I realized that when adding the logo banners for the hyperspy-website, and have adapted it from there.

An alternative would be to use `:class: dark-light` as in https://github.com/matplotlib/mpl-third-party/pull/173

Reply:
Yes, now I can see it! 😃